app/_src/gateway/kong-enterprise/datakit/get-started.md

```diff
@@ -63,55 +63,98 @@ services:
 
 Let's test out Datakit by combining responses from two third party API calls, then returning directly to the client.
 
-In the following example, replace `SERVICE_NAME|ID` with `my-service`, or with your own service name:
-
-<!--vale off-->
-
-{% plugin_example %}
-plugin: kong-inc/acl
-name: datakit
-config:
-  debug: true
-  nodes:
-  - name: CAT_FACT
-    type: call
-    url:  https://catfact.ninja/fact
-  - name: DOG_FACT
-    type: call
-    url:  https://dogapi.dog/api/v1/facts
-  - name: JOIN
-    type: jq
-    inputs:
-    - cat: CAT_FACT.body
-    - dog: DOG_FACT.body
-    jq: |
-      {
-        "cat_fact": $cat.fact,
-        "dog_fact": $dog.facts[0]
-      }
-  - name: EXIT
-    type: exit
-    inputs:
-    - body: JOIN
-    status: 200
-targets:
-  - service
-formats:
-  - curl
-  - konnect
-  - yaml
-  - kubernetes
-  - terraform
-{% endplugin_example %}
-
-<!-- vale on -->
+{% navtabs %}
+{% navtab decK (YAML) %}
+```yaml
+plugins:
+- name: datakit
+  service: my-service
+  config:
+    debug: true
+    nodes:
+    - name: CAT_FACT
+      type: call
+      url: https://catfact.ninja/fact
+    - name: DOG_FACT
+      type: call
+      url: https://dogapi.dog/api/v1/facts
+    - name: JOIN
+      type: jq
+      inputs:
+      - cat: CAT_FACT.body
+      - dog: DOG_FACT.body
+      jq: |
+        {
+          "cat_fact": $cat.fact,
+          "dog_fact": $dog.facts[0]
+        }
+    - name: EXIT
+      type: exit
+      inputs:
+      - body: JOIN
+      status: 200
```

Comment:
```suggestion
plugins:
  - name: datakit
    service: my-service
    config:
      debug: true
      nodes:
      - name: CAT_FACT
        type: call
        url: https://catfact.ninja/fact
      - name: DOG_FACT
        type: call
        url: https://dogapi.dog/api/v1/facts
      - name: JOIN
        type: jq
        inputs:
        - cat: CAT_FACT.body
        - dog: DOG_FACT.body
        jq: |
          {
            "cat_fact": $cat.fact,
            "dog_fact": $dog.facts[0]
          }
      - name: EXIT
        type: exit
        inputs:
        - body: JOIN
        status: 200

      status: 200
```

Reply:
This breaks the config. Maybe it's not pasting correctly from your file? The original decK config in this file works as written for me.